### DISCUSSION TITLE
Why preload does not work in next js 13

### DISCUSSION THREAD

**Discussion by reyangurjar20102008** [vercel/swr#2307] [Q&A] [answered]
I am trying to preload a api but i am getting this

(0 , swr__WEBPACK_IMPORTED_MODULE_2__.preload) is not a function

I am not creating a issue because i think it might make people annoy.

Using Next js 13

**Comment by koba04** [comment, marked as answer]
@reyangurjar20102008 Could you create a CodeSandbox project to reproduce this?

**Reply by reyangurjar20102008** [reply]
@koba04 
https://codesandbox.io/p/github/reyangurjar20102008/nextjs-2bhrdv/draft/thirsty-fast?file=%2FREADME.md

THis is the project

**Reply by koba04** [reply]
@reyangurjar20102008 `preload`was added in v2, but you are using swr@1.3.0. I think you can use `preload` by updating swr v2.

**Reply by reyangurjar20102008** [reply]
Thanks
